app/src/features/surveys/components/MethodForm.tsx

```diff
@@ -41,17 +41,17 @@ export interface IEditSurveySampleMethodData extends ISurveySampleMethodData {
 }
 
 export const SurveySampleMethodPeriodArrayItemInitialValues = {
-  method_lookup_id: null,
-  survey_sample_period_id: null,
-  survey_sample_method_id: null,
+  method_lookup_id: '' as unknown as null,
```

Comment:
Any reason these can't be `null`?

Reply:
(Not sure if this is the issue in this specific case) Lots of values, if they are used in form controls, don't like having a null default. I think the best example is in dropdown controls, they throw a warning/error in the console often if the value is null to start with (rather than empty).